<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateReflectionDialog</class>
 <widget class="QDialog" name="CreateReflectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reflection</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Reflection Plane</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="centerDoubleSpinBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="centerLabel">
        <property name="text">
         <string>Center</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="xMinRadioButton">
        <property name="text">
         <string>XMin</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="yMinRadioButton">
        <property name="text">
         <string>YMin</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="zMinRadioButton">
        <property name="text">
         <string>ZMin</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="xMaxRadioButton">
        <property name="text">
         <string>XMax</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="xRadioButton">
        <property name="text">
         <string>X</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="yMaxRadioButton">
        <property name="text">
         <string>YMax</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="zMaxRadioButton">
        <property name="text">
         <string>ZMax</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="yRadioButton">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QRadioButton" name="zRadioButton">
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateReflectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateReflectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
